extensions/pubsub/standard/sections/annex-pubsub.adoc

```diff
@@ -142,6 +142,8 @@ Different Pub/Sub protocols are supported as additional server objects, and can
 ** `collections/wthr_stn/items`
 ** `collections/stream_gage/items`
 
+The use of `x-ogc-api-link` demonstrates the capability to cross reference an AsyncAPI to an OGC API resource.
```

Comment:
The same concerns as in #451 apply here as well.

`rel` is used incorrectly and are not consistent with the definitions of the link relation type. The link relations `collection` or `items` are not applicable to the context of the link (a channel).

The use of `type` is also unclear to me.

My guess is that this construct is intended to identify a resource where a change in the resource results in a message and that also the type of the resource should be declared. The current approach as I understand it from the examples and that still is not specified, does not provide a solution for this.